metricbeat/module/system/process/config.go

```diff
@@ -32,6 +32,8 @@ type Config struct {
 	IncludeCPUTicks bool                     `config:"process.include_cpu_ticks"`
 	IncludePerCPU   bool                     `config:"process.include_per_cpu"`
 	CPUTicks        *bool                    `config:"cpu_ticks"` // Deprecated
+	// Pid, if set, will override the `processes` config, and only monitor a single process.
+	Pid int `config:"process.pid"`
```

Comment:
Not a blocker, but what if the config field was just `pid` instead of `process.pid`? It seems a little confusing having two mutually exclusive top-level fields to specify the target, but one of them lives in a subobject for other behavior settings.